great_expectations_provider/example_dags/example_great_expectations_dag.py

```diff
@@ -3,113 +3,208 @@
+                    "module_name": "great_expectations.execution_engine",
+                    "class_name": "PandasExecutionEngine",
+                },
+                "class_name": "Datasource",
+            }
+        },
+        "config_variables_file_path": os.path.join(
+            ge_root_dir, "uncommitted", "config_variables.yml"
+        ),
+        "stores": {
+            "expectations_store": {
+                "class_name": "ExpectationsStore",
+                "store_backend": {
+                    "class_name": "TupleFilesystemStoreBackend",
+                    "base_directory": os.path.join(ge_root_dir, "expectations"),
+                },
+            },
+            "validations_store": {
+                "class_name": "ValidationsStore",
+                "store_backend": {
+                    "class_name": "TupleFilesystemStoreBackend",
+                    "base_directory": os.path.join(
+                        ge_root_dir, "uncommitted", "validations"
+                    ),
+                },
+            },
+            "evaluation_parameter_store": {"class_name": "EvaluationParameterStore"},
+            "checkpoint_store": {
+                "class_name": "CheckpointStore",
+                "store_backend": {
+                    "class_name": "TupleFilesystemStoreBackend",
+                    "suppress_store_backend_id": True,
+                    "base_directory": os.path.join(ge_root_dir, "checkpoints"),
+                },
+            },
+        },
+        "expectations_store_name": "expectations_store",
+        "validations_store_name": "validations_store",
+        "evaluation_parameter_store_name": "evaluation_parameter_store",
+        "checkpoint_store_name": "checkpoint_store",
+        "data_docs_sites": {
+            "local_site": {
+                "class_name": "SiteBuilder",
+                "show_how_to_buttons": True,
+                "store_backend": {
+                    "class_name": "TupleFilesystemStoreBackend",
+                    "base_directory": os.path.join(
+                        ge_root_dir, "uncommitted", "data_docs", "local_site"
+                    ),
+                },
+                "site_index_builder": {"class_name": "DefaultSiteIndexBuilder"},
+            }
+        },
+        "anonymous_usage_statistics": {
+            "data_context_id": "abcdabcd-1111-2222-3333-abcdabcdabcd",
+            "enabled": False,
+        },
+        "notebooks": None,
+        "concurrency": {"enabled": False},
+    }
 )
 
-# This runs an expectation suite against a data asset that passes the tests
-ge_batch_kwargs_list_pass = GreatExpectationsOperator(
-    task_id="ge_batch_kwargs_list_pass",
-    assets_to_validate=[
-        {
-            "batch_kwargs": {"path": data_file, "datasource": "data__dir"},
-            "expectation_suite_name": "taxi.demo",
-        }
-    ],
-    data_context_root_dir=ge_root_dir,
-    dag=dag,
+checkpoint_config = CheckpointConfig(
```

Comment:
I think this and the `DataContextConfig` should go in a static config file under `include/`, no need to clutter up the DAG file!